### ISSUE TITLE
Overridden `.send` command doesn't work in discord.py master branch (v2) 

### ISSUE THREAD

**Issue by noaione** [interactions-py/interactions.py#287]
Apparently, discord-slash [overrides](https://github.com/discord-py-interactions/discord-py-interactions/blob/master/discord_slash/dpy_overrides.py) some of the original discord.py method and since I'm using the master v2 branch, the command broke if I'm trying to add `view` kwargs since the kwargs doesn't exist in the overrides.

Traceback:
```py
[2021-08-11 22:33:47 REDACTED][ERROR] (REDACTED[20332]): on_command_error: Ignoring exception in command kbbi:
[2021-08-11 22:33:47 REDACTED][ERROR] (REDACTED[20332]): echo_error: Exception occured
Traceback (most recent call last):
  File "[REDACTED]\env\lib\site-packages\discord\ext\commands\core.py", line 125, in wrapped
    ret = await coro(*args, **kwargs)
  File "[REDACTED]\cogs\kutubuku\kbbi.py", line 478, in _kbbi_cmd_main
    await ctx.send(embed=first_embed, view=ui_paginate)
  File "[REDACTED]\env\lib\site-packages\discord_slash\dpy_overrides.py", line 323, in send_override
    return await send(channel, *args, **kwargs)
TypeError: send() got an unexpected keyword argument 'view'
```

**Version used:**
Discord.py: https://github.com/Rapptz/discord.py/commit/58ca9e99325ac2678a51cd63afd7ae917f14bc8d
discord-slash: 2.4.1

**Comment by i0bs** [comment]
This has been internally discussed with the contributors a short time back but it is to be expected for d.py 2.0 to generally bring compatibility issues to our library. Our library supports only 1.7.3 and below at this time because the new discord.py 2.0 logic affects the way we're able to make things work. Our overrides are a small monkeypatch to keep stability for the most part between both worlds.

**Comment by noaione** [comment]
I'm guessing, if I try to monkey-patch again the function (maintaining both dpy and dslash difference) it would work just fine?

**Comment by noaione** [comment]
After tinkering for a while, this works for me:
https://gist.github.com/noaione/738ce5ba570b4d27e00f8e69679b0bff

I monkey-patched it again after binding `SlashCommand` to the bot.

**Comment by i0bs** [comment]
I'm moving this to be a wontfix because this is going to be addressed personally as-is in v4.0 by not supporting the `view` kwarg. That's an arbitrary field that was created for discord.py 2.0 that I don't see a need to also support since we will be heading to become an independent API wrapper.
